opmon-protobuf-dbwriter/opmon-dbwriter-deployment.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    kluctl.io/skip-delete-if-tags: "true"
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/enforce: baseline # unified image runs as root :(
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/warn-version: latest
+  name: ers
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/app: opmon-protobuf-dbwriter
+    app.kubernetes.io/component: opmon-protobuf-dbwriter
+  name: opmonstream-dbwriter
+  namespace: opmon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: opmon-protobuf-replica
```

Comment:
I generally try to have `matchLabels` match up with the `template.metadata.labels`

Reply:
That I'm not sure how to address it, I made the names more uniform. Let me know if this makes sense.